.github/workflows/main.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  shellcheck:
+    name: Shellcheck
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Run ShellCheck
+        uses: ludeeus/action-shellcheck@master
+        with:
+          severity: info
+          scandir: 'bin/'
+        env:
+          SHELLCHECK_OPTS: -s sh -e SC1091 -e SC2034 -e SC2317
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Make ECM Distro Tools
+        run: |
+          make all
```

Comment:
are we concerned with go version, or are we building within docker?

Reply:
Not concerned with the version of Go.